integrationtests/migration/Makefile

```diff
@@ -0,0 +1,12 @@
+.EXPORT_ALL_VARIABLES:
+
+RUN_MIGRATION_TEST="true"
+ENV_FILENAME="thread_ids"
+ENV_NUM_RECORDS="10"
+ENV_NUM_THREADS="10"
+
+test-migration:
```

Comment:
So I've found out that implementing the migration test in testground is actually not convenient. Because you need to have:
- two different docker containers versions pointing to two different repositories
- they should have the same storage (because how would the other one migrate on the results of the first one)
- we need to also note that the head structure is different, so these containers should not use the same test (see `GetHead` method in test or we should create some helper methods outside the test) which is not as easy as it seems

So instead I decided to go with the quick and dirty solution of just generating the records with the old version and the having the test using the other version which seems to work pretty well.

Reply:
makes total sense esp. the latter two points, and this test is much easier to run than having to spin up testground